### DISCUSSION TITLE
Error in Experiments with start_time# example

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3087] [Q&A]
There was an error when running the following code in the example: [https://docs.pybamm.org/en/latest/source/examples/notebooks/experiments-start-time.html](url)

s = pybamm.step.string

experiment = pybamm.Experiment(
    [
        s("Discharge at 1C for 1 hour", start_time=datetime(1, 1, 1, 8, 0, 0)),
        s("Charge at C/3 for 10 minutes", start_time=datetime(1, 1, 1, 8, 30, 0)),
        s("Discharge at C/2 for 30 minutes", start_time=datetime(1, 1, 1, 9, 0, 0)),
        s("Rest for 1 hour"),
    ]
)
sim = pybamm.Simulation(model, experiment=experiment)
sim.solve()
sim.plot()

The error is: No attribute call step in PyBaMM

Can someone provide me reason for this?

**Comment by brosaplanella** [comment]
You are running an old version of PyBaMM (i.e. older than 23.5), but using the example for version 23.5.
